target_chains/ton/contracts/contracts/tests/PythTest.fc

```diff
@@ -0,0 +1,27 @@
+#include "../imports/stdlib.fc";
+#include "../common/data.fc";
+#include "../Pyth.fc";
+
+() recv_internal(int balance, int msg_value, cell in_msg_full, slice in_msg_body) impure {
+    ;; nop;
+}
+
+(int, int, int, int) test_price_unsafe(int price_feed_id) method_id {
```

Comment:
I'm a bit confused about these methods. What happens if an external caller calls get_price when the global state is not loaded?

Reply:
then the variables will be uninitialized `()` and an error will be thrown

Reply:
so how possibly an external caller can call us :?